### PR TITLE
feat(task): Add API events for drawtask plugin

### PR DESCRIPTION
**Pull request by Cerzus**
Add api.tasks.on.drawBegin, api.tasks.on.draw and
api.tasks.on.drawEnd events for the drawtask plugin.

### CODE REVIEW

File: demo/app/scripts/controllers/main.js [hunk 1]
```diff
@@ -204,6 +204,12 @@ angular.module('angularGanttDemoApp')
                         api.tasks.on.resizeEnd($scope, addEventName('tasks.on.resizeEnd', logTaskEvent));
                     }
 
+                    if (api.tasks.on.drawBegin) {
+                        api.tasks.on.drawBegin($scope, addEventName('tasks.on.drawBegin', logTaskEvent));
+                        //api.tasks.on.draw($scope, addEventName('tasks.on.draw', logTaskEvent));
```
Comment (Toilal) [inline demo/app/scripts/controllers/main.js]: I think it should be uncommented, isn't it ?

Comment (Toilal) [inline demo/app/scripts/controllers/main.js]: Or simply removed ?

**Comment by Cerzus** [comment]
I added this feature because in my project I need to remove tasks with zero length (tasks with equal from and to moments). At first I tried to listen to the resizeEnd event, but it turns out that that event is not fired when the mouse has moved exactly enough to pass the threshold.

I figured the drawtask plugin could use its own events just like the movable plugin.

**Comment by Toilal** [comment]
Thank you ! I'll give it a try a soon as possible.

File: demo/app/scripts/controllers/main.js (hunk 1 above)
Comment (Cerzus) [inline demo/app/scripts/controllers/main.js]: It's commented out just like the resize and move events of the movable plugin. Why are those commented out?

Comment (Toilal) [inline demo/app/scripts/controllers/main.js]: Ho yes, just realized it's in the demo sources. Lets keep those comments, people reading sources can see thos events exists, but we don't want to flood the console with logs.

Comment (Cerzus) [inline demo/app/scripts/controllers/main.js]: Exactly :)

**Comment by Schweigi** [comment]
@Cerzus Thx for the PR. It works great.

**Pull request merged**